tree/dataframe/test/dataframe_helpers.cxx

```diff
@@ -275,3 +279,66 @@ TEST(RDFHelpers, SaveGraphToFile)
 
    gSystem->Unlink(outFileName);
 }
+
+TEST(RunGraphs, RunGraphs)
+{
+#ifdef R__USE_IMT
+   ROOT::EnableImplicitMT();
+#endif // R__USE_IMT
```

Comment:
do we have any builds with `-Dimt=OFF`? i.e. will the non-MT case ever be tested?

Reply:
Yep, we have two (?) builds with IMT off

Reply:
ok, `-Dimt=ON` + single-thread will never be tested but that's ok, `-Dimt=OFF` should be pretty much the same, ideally